app/css/lists.css

```diff
@@ -8,3 +8,9 @@
     width: 100%;
     text-decoration: none;
 }
+
+[data-type="budyup-list"] header {
+  color: #0AC;
+  text-align: left;
+  padding-left: 0.7rem;
```

Comment:
I'm not very focused on spacing and things like that yet. We'll receive the visuals next week with the metrics and everything so this is when we can fix this kind of stuff.